File: LeetCodeDSA/2027.c
#include <stdio.h>
#include <string.h>

int main(){
    char* s = "XXX"; 
    int lenght = strlen(s); 
    int count = 0; 
    int i = 0; 
    while(i < lenght){
        if(s[i] == 'X'){
            count += 1; 
            i += 3; 
        }else{
            i += 1;
        }
    }
    printf("%d", count); 
    return 0;
}

File: LeetCodeDSA/2027.py
def minimumMoves(s):
    lenght = len(s) - 1
    count = 0
    i = 0
    while i <= lenght:
        if s[i] == 'X':
            count += 1
            i += 3
        else: 
            i += 1
    return count    
print(minimumMoves(s = "XXOX"))